src/style.css

```diff
@@ -1,3 +1,66 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+background-color: gainsboro;
+}
+
+body, .header_nav a {
+  font-family: 'Roboto', sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, `font-family: 'Roboto', sans-serif;` should include a generic font family like `sans-serif` as a fallback.